configuration.yaml

```diff
@@ -10,4 +10,4 @@ admin:
   mail: admin@localhost
   password: mypassword
 secretkey: JustWriteSomethingWhatYouLike
-address: :8080
+address: :8081
```

Comment:
Hey that is a good job. But the configuration, better not commit

Reply:
Opps, my 8080 port was blocked

Reply:
Should be fixed by now, check